tests/autotuner/test_autotuner_core.py

```
@@ -838,3 +838,180 @@ def test_choose_one_with_none_input_no_crash():
+    inputs = [torch.empty((16, 32), dtype=torch.float32)]
+    config = TuningConfig()
+
+    def fake_profile(
+        self, runner_obj, prof_inputs, tactic, tuning_config=None, **kwargs
+    ):
+        return {0: 5.0, 1: 1.0}[tactic]
+
+    monkeypatch.setattr(AutoTuner, "_profile_single_kernel", fake_profile)
+
+    with autotune(tune_mode=True, skip_ops=set()):
+        chosen, tactic = tuner.choose_one("should_tune", [runner], config, inputs)
```

Ruff (0.15.14)

[warning] Unpacked variable `chosen` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)